packages/fxa-auth-server/test/local/routes/subscriptions/stripe-webhooks.js

```diff
@@ -486,6 +511,97 @@ describe('StripeWebhookHandler', () => {
       });
     });
 
+    describe('handleProductUpdatedEvent', () => {
+      let scopeContextSpy, scopeSpy;
+      beforeEach(() => {
+        scopeContextSpy = sinon.fake();
+        scopeSpy = {
+          setContext: scopeContextSpy,
+        };
+        sandbox.replace(Sentry, 'withScope', (fn) => fn(scopeSpy));
+      });
+
+      it('throws a sentry error if the update event data is invalid', async () => {
```

Comment:
Since a product update triggers multiple validations, we should verify the correct number of errors are reported.

I also don't see that the tests verifying that Sentry.captureMessage has been called.